www/api/modules/payments/gateways/mercadopago/domain/service/card/card_service.py

```diff
@@ -43,7 +42,7 @@ def pay(
             "payment_method_id": payment_method_id,
             "installments": 1,
             "payer": {
-                "email": self.user_service.encrypt_email(req_user.email)
+                "email": req_user.email
```

Comment:
You need to add back the encryption we don't want to share actual user emails with mercadolibre

Reply:
this return me error from mercado pago, when i remove this its make the payment

Reply:
Okay then please add a TODO:"fix hidden mail" and send a hardcoded Gmail fake mail instead of the user mail